src/main/java/core/basesyntax/dao/WarehouseDaoWrite.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class WarehouseDaoWrite {
+    public static final String BEGIN_OF_CSV = "fruit,quantity";
+
+    public static void writeData(String fileName, Map<String, Integer> remains) {
+        List<String> outputList = remains.entrySet().stream().map(
+                entry -> entry.getKey() + "," + entry.getValue()).collect(Collectors.toList());
+        Path filePath = Paths.get(fileName);
+        try {
```

Comment:
Instead of deleting and creating a new file, you can use 'Files.write' with 'StandardOpenOption.TRUNCATE_EXISTING' or 'StandardOpenOption.CREATE' to either overwrite or create a file if it doesn't exist.